config/unicorn.rb

```diff
@@ -26,7 +36,7 @@
   end
 end
 
-after_fork do |_server, _worker|
+after_fork do
```

Comment:
Please revert this line: `after_fork=#<Proc:0x00000002308a48@config/unicorn.rb:39> has invalid arity: 0 (need 2) (ArgumentError)`